tests/integ/test_udf.py

```diff
@@ -1937,6 +1937,29 @@ def check_len(s):
     )
 
 
+# Test fix for SNOW-836583, when the file does not explicitly mark the handler function as vectorized UDF
+def test_register_pandas_udf_without_specification(session, tmpdir):
+    source = """
+def add_one_to_inputs(df):
+  return df[0] + df[1] + 1
+"""
+    file_path = os.path.join(
+        tmpdir, "test_register_pandas_udf_without_specification.py"
+    )
+    with open(file_path, "w") as f:
+        f.write(source)
```

Comment:
we could also add a file in `tests/resources/test_udf_dir` and reuse it for more tests